.github/workflows/ci.yaml

```diff
@@ -19,4 +23,36 @@ jobs:
 
     - name: Run tests
       run: |
-        uv run pytest tests -m 'not integration'
\ No newline at end of file
+        uv run pytest tests -m 'not integration'
+
+  create_tag:
```

Comment:
should this be run on the release workflow - not ci? ci just for testing?

Reply:
I can move this on the release workflow.  and the release workflow depends on the ci workflow and only run on the main branch

Reply:
I've pushed a commit to move the `create_tag` job to the `pypi_release` workflow now